openwisp_controller/config/admin.py

```diff
@@ -91,7 +91,7 @@ def get_extra_context(self, pk=None):
             ctx['download_url'] = reverse('{0}_download'.format(prefix), args=[pk])
             try:
                 has_config = (
-                    self.model.__name__ == 'Device'
+                    self.model.__name__ in ['Device', 'Template', 'Vpn']
```

Comment:
We can simplify this.

`_has_config` does not have any sense on `Template` and `Vpn` because they always have a config. A `Device` may not.
Something like the following code is enough:

```python
has_config = True
if self.model.__name__ == 'Device':
    has_config = self.model.objects.get(pk=pk)._has_config()
```

Please keep in mind to look for simple solutions. Do not change the models just to avoid changing an if, just change the if and avoid adding code to the models.